addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -133,8 +133,74 @@
+                    <we-button data-visibility-comparaison="!set">Is not set</we-button>
+                </we-select>
+                <we-select data-name="hidden_condition_date_opt" data-no-preview="true">
+                    <!-- date comparaison possibilities -->
+                    <we-button data-visibility-comparaison="equal">Is equal to</we-button>
+                    <we-button data-visibility-comparaison="!equal">Is not equal to</we-button>
+                    <we-button data-visibility-comparaison="after">Is after</we-button>
+                    <we-button data-visibility-comparaison="before">Is before</we-button>
+                    <we-button data-visibility-comparaison="equal or after">Is after or equal to</we-button>
+                    <we-button data-visibility-comparaison="equal or before">Is before or equal to</we-button>
+                    <we-button data-visibility-comparaison="set">Is set</we-button>
+                    <we-button data-visibility-comparaison="!set">Is not set</we-button>
+                    <we-button data-visibility-comparaison='between'>Is between (included)</we-button>
+                    <we-button data-visibility-comparaison='!between'>Is not between (excluded)</we-button>
+                </we-select>
+                <we-select data-name="hidden_condition_datetime_opt" data-no-preview="true">
+                    <!-- dateTime comparaison possibilities -->
+                    <we-button data-visibility-comparaison="equal">Is equal to</we-button>
+                    <we-button data-visibility-comparaison="!equal">Is not equal to</we-button>
+                    <we-button data-visibility-comparaison="after">Is after</we-button>
+                    <we-button data-visibility-comparaison="before">Is before</we-button>
+                    <we-button data-visibility-comparaison="equal or after">Is after or equal to</we-button>
+                    <we-button data-visibility-comparaison="equal or before">Is before or equal to</we-button>
+                    <we-button data-visibility-comparaison="set">Is set</we-button>
+                    <we-button data-visibility-comparaison="!set">Is not set</we-button>
+                    <we-button data-visibility-comparaison='between'>Is between (included)</we-button>
+                    <we-button data-visibility-comparaison='!between'>Is not between (excluded)</we-button>
+                </we-select>
+            </we-row>
+            <we-select class="o_we_large_select" data-name="hidden_condition_no_text_opt" data-no-preview="true">
+                <!-- checkbox, select, radio possible values -->
+            </we-select>
+            <we-input class="o_we_large_input" data-name="hidden_condition_additional_text" data-visibility-condition="" />
+            <we-datetimepicker class="o_we_large_input" data-name="hidden_condition_additional_datetime" data-visibility-condition="" />
```

Comment:
Is the `o_we_large_input` actually needed with date widgets? Is not this automatic? If not, it should :wink: